CHANGELOG.rst

```diff
@@ -1,5 +1,29 @@
 .. _changelog:
 
+0.12.3
+------
+
+Minor release coming with:
+* several new features on renku-ui,
+* new /knowledge-graph/entities (cross-entity search) API,
+* fixes around Cross-Origin Resource Sharing,
+* better messaging and reporting on renku-notebooks
+* fixes for various bugs in `renku-notebooks`, `renku-ui`, `renku-gateway` and `graph` services.
```

Comment:
```suggestion
* fixes for various bugs in renku-notebooks, renku-ui, renku-gateway and renku-graph services.
```

For consistency, I would either remove the formatting here or add it also above where you mention the renku services